lib/amazon/parser/flavor.rb

```diff
@@ -48,7 +47,8 @@ def parse_flavor_storage(storage)
       end
 
       def parse_flavor_memory(memory)
-        match = /^((\d*[.])?(\d+)).*$/.match(memory.gsub(",", ""))
+        match = /^((\d*[.])?(\d+)).*$/.match(memory&.gsub(",", ""))
+        return 0 unless match
```

Comment:
@ladas I feel like this should return nil, otherwise this means literally 0MB of memory vs "we don't know the memory value"

Reply:
hm you're probably right

Reply:
done